retroshare.pri

```diff
@@ -38,8 +38,8 @@ no_retroshare_nogui:CONFIG -= retroshare_nogui
 
 # To disable cmark append the following 
 # assignation to qmake command line "CONFIG+=no_cmark"
-CONFIG *= no_cmark
-cmark:CONFIG -= no_cmark
+CONFIG *= cmark
+no_cmark:CONFIG -= cmark
```

Comment:
If it is a retroshare-gui specific stuff the config name should reflect it for example rs_gui_cmark

Reply:
OK